src/Castle.Windsor/Windsor/Installer/FromAssembly.cs

```diff
@@ -115,12 +115,19 @@ public static IWindsorInstaller InDirectory(AssemblyFilter filter, InstallerFact
 		///     <c>MyApp.SuperFeatures.dll</c>, <c>mscorlib.dll</c> and <c>ThirdPartyCompany.UberControls.dll</c> the <c>MyApp.exe</c> and <c>MyApp.SuperFeatures.dll</c> will be scanned for installers, and other
 		///     assemblies will be ignored.
 		/// </remarks>
+#if FEATURE_GETCALLINGASSEMBLY
 		[MethodImpl(MethodImplOptions.NoInlining)]
 		public static IWindsorInstaller InThisApplication()
 		{
 			var assembly = Assembly.GetCallingAssembly();
 			return ApplicationAssemblies(assembly, new InstallerFactory());
```

Comment:
Could we change the implementation of this method to just call the new one below and pass `GetCallingAssembly()`.